internal_build.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-sh -$- build.sh baidu
+source ./build.sh baidu
```

Comment:
so that build.sh can inherit the -x option set by user
also inherit the shell bin (bash/csh/...) set by user